docs/additional-functionality/rapids-udfs.md

```diff
@@ -33,16 +33,20 @@ on the GPU.
 The RAPIDS Accelerator only supports RAPIDS accelerated forms of the
 following UDF types:
 - Scala UDFs implementing a `Function` interface and registered via `SparkSession.udf.register`
+- Java UDFs implementing
+  [one of the `org.apache.spark.sql.api.java.UDF` interfaces](https://github.com/apache/spark/tree/branch-3.0/sql/core/src/main/java/org/apache/spark/sql/api/java)
```

Comment:
referencing branch-3.0 here, is that what you want?

Reply:
Yes, in case Apache Spark decides to change it later or renames the `master` branch and the link ends up dangling.